changelog/4524.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+All calculations of the angular radius of the Sun now use the same underlying code with the correct calculation.
+The inaccuracies were at the level of ~0.01 arcseconds.
+A previous fix to :func:`sunpy.coordinates.sun.angular_radius` was incorrect and has been reverted.
```

Comment:
You can delete the other changelog entry. No need to confuse it.

Reply:
Sure, but should I remove this sentence too then?  It may be confusing to refer to a "previous fix" when there is no corresponding changelog entry.

Reply:
yeah, I feel like we should be writing the changelog to be read at release time, we don't really need to document release dev fluctuations in it.

Reply:
Okay, done